api/tests/test_user_roles.py

```diff
@@ -0,0 +1,162 @@
+from flask_bcrypt import Bcrypt
+
+from ..user_roles import *
```

Comment:
See if you can remove this relational import, you should just be able to import 'user_roles'. ^^^